assignments/lambda-classes.js

```diff
@@ -1 +1,114 @@
 // CODE here for your Lambda Classes
+class Person {
+  constructor(attributes) {
+    this.gender = attributes.gender;
+    this.age = attributes.age;
+    this.name = attributes.name;
+    this.location = attributes.location;
+  }
+  speak() {
+    return `Hello, my name is ${this.name} and Im from ${this.location}`;
+  }
+}
+
+class Instructor extends Person {
+  constructor(instructorAttrs) {
+    super(instructorAttrs);
+    this.gender = instructorAttrs.gender;
+    this.age = instructorAttrs.age;
+    this.name = instructorAttrs.name;
+    this.location = instructorAttrs.location;
+    this.isInstructor = instructorAttrs.isInstructor;
+    this.specialty = instructorAttrs.specialty;
```

Comment:
What is the usefulness of this property? It doesn't look like you assigned anything to it below when you created your objects. If you are trying to verify if an object has a particular class, there are built in methods for that sort of thing. [instanceof](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof)